UICatalog/Scenarios/CsvEditor.cs

```diff
@@ -394,7 +394,7 @@ private void Save ()
 		private void Open ()
 		{
 			var ofd = new FileDialog () {
-				AllowedTypes = new List<AllowedType> { new AllowedType("Comma Separated Values (.csv)", ".csv") }
+				AllowedTypes = new List<AllowedType> { new AllowedType("Comma Separated Values", ".csv") }
```

Comment:
I think that is needed, at least before was, because the first parameter show the information to the user and the second parameter will provide the extension to the source. Sorry if I'm wrong.

Reply:
I will revisit this before taking this out of Draft.  

I want to look to empower `AllowedType` as well as not break the old API for OpenDialog/SaveDialog.

Ultimately user should be able to say things like:

- Only show/let pick a file called `Dockerfile`

But also I like the simplicity of the old API so that will be kept when going through OpenFile or SaveFile subclasses.

By making `AllowedType` a first class citizen then user can subclass or tailor as needed and many cool things will be possible.